II.pass-tests/ES-6-practice/es6.js

```diff
@@ -1,28 +1,23 @@
 // Turn this function into an arrow function
```

Comment:
Because I missed last week's class I'm not 100% sure if you covered ES6. If you didn't we will look at this in the future, so don't worry if it's confusing.